code/game/objects/structures/crates_lockers/med_crate.dm

```diff
@@ -9,7 +9,7 @@
 		/obj/item/stack/medical/advanced/bruise_pack = 10,
 		/obj/item/chems/pill/sugariron = 6,
 		/obj/item/storage/pill_bottle/painkillers = 2,
-		/obj/item/storage/pill_bottle/adrenaline
+		/obj/item/storage/pill_bottle/stabilizer
```

Comment:
Looks like another chem mixup

Reply:
Also adrenaline pills make no sense mechanically anyways, since injested reagents aren't processed when a mob is dead.

Reply:
Yeah I guess this is meant to be stabilizer